### PR TITLE
Remove 'wiki_settings' and 'wiki_extensions' from cw_wikis

### PR DESCRIPTION
**Pull request by paladox**
Both are unused and were moved to mw_settings (ManageWiki)

### CODE REVIEW

File: maintenance/DBListGenerator.php [hunk 1]
```diff
@@ -33,16 +33,30 @@ public function execute() {
...
-			$private = $row->wiki_private;
-			$closed = $row->wiki_closed;
-			$inactive = $row->wiki_inactive;
-			$deleted = $row->wiki_deleted;
-			$extensions = $row->wiki_extensions;
-			$settings = $row->wiki_settings;
-
-			if ( $deleted === "0" ) {
+
+			if ( (int)$deleted === "0" ) {
```
Comment (JohnFLewis) [inline maintenance/DBListGenerator.php:37]: Always false

File: maintenance/DBListGenerator.php [hunk 2]
```diff
@@ -33,16 +33,30 @@ public function execute() {
...
+				$siteName = $row->wiki_sitename;
+				$language = $row->wiki_language;
+				$private = $row->wiki_private;
+				$closed = $row->wiki_closed;
+				$inactive = $row->wiki_inactive;
+				$deleted = $row->wiki_deleted;
+
+				$row = $dbw->selectRow(
```
Comment (JohnFLewis) [inline maintenance/DBListGenerator.php:45]: This would mean CreateWiki requires ManageWikk

File: maintenance/DBListGenerator.php (hunk 1 above)
Comment (paladox) [reply maintenance/DBListGenerator.php:45]: Thanks, done.

**Comment by DarkMatterMan4500** [comment]
Sorry to interrupt, but for some reason, I got an error on Crappy Games Wiki saying that the page isn't working.

**Comment by dmehus** [comment]
> Sorry to interrupt, but for some reason, I got an error on Crappy Games Wiki saying that the page isn't working.

Please raise this on IRC or Discord with system administrators, quoting the specific error. This isn't appropriate for GitHub.

**Comment by DarkMatterMan4500** [comment]
I raised it on Discord raising this concern, so I now leave this thread to rest.

**Pull request closed without merging**